rpm/common/Makefile

```diff
@@ -0,0 +1,40 @@
+NAME          = archivematica-common
+VERSION       = 1.4.1
```

Comment:
FYI next release will be `1.5.0`.
Maybe we should work on a pre-release to avoid confusions.
https://fedoraproject.org/wiki/Packaging:NamingGuidelines#Package_Versioning